modules/bitgo/package.json

```diff
@@ -94,6 +94,7 @@
     "fs-extra": "^9.1.0",
     "lodash": "^4.17.14",
     "openpgp": "5.1.0",
+    "secp256k1": "^4.0.2",
```

Comment:
Let's not add another secp256k1 library. We already have a secp256k1 library (`noble-secp256k1`) in the SDK, please use it.

`import { ecc as secp256k1 } from '@bitgo/utxo-lib';`